sa/model.go

```diff
@@ -15,6 +16,10 @@ import (
 	"github.com/letsencrypt/boulder/revocation"
 )
 
+// By convention, any function that takes one of the following as arguments
```

Comment:
Can you update this comment to explicitly reference `dbOneSelector`, `dbInserter`, and `dbExecer`. At the moment it's just hanging and it's not entirely clear what it's talking about.